platforms/native/language-native/src/integTest/groovy/org/gradle/language/swift/SwiftApplicationDependenciesIntegrationTest.groovy

```diff
@@ -19,8 +19,10 @@ package org.gradle.language.swift
 import org.gradle.language.AbstractNativeProductionComponentDependenciesIntegrationTest
 import org.gradle.nativeplatform.fixtures.RequiresInstalledToolChain
 import org.gradle.nativeplatform.fixtures.ToolChainRequirement
+import org.gradle.test.fixtures.file.DoesNotSupportNonAsciiPaths
 
 @RequiresInstalledToolChain(ToolChainRequirement.SWIFTC)
+@DoesNotSupportNonAsciiPaths(reason = "swiftc does not support these paths")
```

Comment:
New versions of the Swift tooling seem to blow up when there are non-ascii paths. For now, ignore this and use ascii paths.